Translations/Polish/README.md

```diff
@@ -146,9 +146,13 @@ Other scopes![other scopes](http://stackoverflow.com/a/500459)
 * Jaka jest różnica między `==` i `===`?
 * Wyjaśnij ewentualny sposób pobrania parametrów z adresu URL w oknie przeglądarki.
 * Wyjaśnij politykę `same-origin` w odniesieniu do JavaScript.
+    Jest to fundamentalny mechanizm bezpieczeństwa przeglądarki. Mechanizm ten zapobiega dostępowi
+    skryptów js różnego pochodzenia do drzew DOM znajdujących się na różnych originach.
```

Comment:
nie nie, chodziło mi o użycie tego przy różnych orignach po polsku to: różnego pochodzenia ;)